### PR TITLE
Troubleshooting follower reads on global tables

### PR DESCRIPTION
**Pull request by rmloveland**
Fixes DOC-6501

Summary of changes:

- Update 'Global Tables' page with a 'Troubleshooting > High follower read latency on global tables' section that describes the problem and solution

- Add more linkage from 'Multiregion Overview' to 'Global Tables', so people can find this information

- Add more linkage from 'Follower Reads' to 'Global Tables' so people can find this

### CODE REVIEW

File: v21.2/global-tables.md [hunk 1]
```diff
@@ -84,6 +84,14 @@ Because the `test` database does not specify a [survival goal](multiregion-overv
 
 For more information about how to choose a database survival goal, see [When to Use `ZONE` vs. `REGION` Survival Goals](when-to-use-zone-vs-region-survival-goals.html).
 
+## Troubleshooting
+
+### High follower read latency on global tables
+
+Reads on multiregion global tables can experience sporadic high latency on [follower reads](follower-reads.html) if the round trip time between cluster nodes is higher than 150ms. To work around this issue, consider setting the `kv.closed_timestamp.lead_for_global_reads_override` [cluster setting](cluster-settings.html) to a value greater than 800ms.
```
Comment (andrewbaptist) [inline v21.2/global-tables.md:91]: Just the comment that github added - change this from multiregion to multi-region

**Review by andrewbaptist — APPROVED** [review]

File: v21.2/global-tables.md (hunk 1 above)
Comment (rmloveland) [reply v21.2/global-tables.md]: thanks Andrew - pushed a fix!

File: v21.2/global-tables.md [hunk 2]
```diff
@@ -84,6 +84,14 @@ Because the `test` database does not specify a [survival goal](multiregion-overv
 
 For more information about how to choose a database survival goal, see [When to Use `ZONE` vs. `REGION` Survival Goals](when-to-use-zone-vs-region-survival-goals.html).
 
+## Troubleshooting
+
+### High follower read latency on global tables
+
+reads on multi-region global tables can experience sporadic high latency on [follower reads](follower-reads.html) if the round trip time between cluster nodes is higher than 150ms. To work around this issue, consider setting the `kv.closed_timestamp.lead_for_global_reads_override` [cluster setting](cluster-settings.html) to a value greater than 800ms.
```
Comment (ianjevans) [inline v21.2/global-tables.md:91]:
```suggestion
Reads on multi-region global tables can experience sporadic high latency on [follower reads](follower-reads.html) if the round trip time between cluster nodes is higher than 150ms. To work around this issue, consider setting the `kv.closed_timestamp.lead_for_global_reads_override` [cluster setting](cluster-settings.html) to a value greater than 800ms. 
```

File: v21.2/global-tables.md [hunk 3]
```diff
@@ -84,6 +84,14 @@ Because the `test` database does not specify a [survival goal](multiregion-overv
 
 For more information about how to choose a database survival goal, see [When to Use `ZONE` vs. `REGION` Survival Goals](when-to-use-zone-vs-region-survival-goals.html).
 
+## Troubleshooting
+
+### High follower read latency on global tables
+
+reads on multi-region global tables can experience sporadic high latency on [follower reads](follower-reads.html) if the round trip time between cluster nodes is higher than 150ms. To work around this issue, consider setting the `kv.closed_timestamp.lead_for_global_reads_override` [cluster setting](cluster-settings.html) to a value greater than 800ms. 
+
+Note that this value will impact write latency to global tables, so you should proceed in 100ms increments until the high read latency no longer occurs. If you've increased the setting to 1500ms and the problem persists, you should [contact support](support-resources.html).
```
Comment (ianjevans) [inline v21.2/global-tables.md:93]:
```suggestion
The value of `kv.closed_timestamp.lead_for_global_reads_override` will impact write latency to global tables, so you should proceed in 100ms increments until the high read latency no longer occurs. If you've increased the setting to 1500ms and the problem persists, you should [contact support](support-resources.html).
```

File: v22.1/global-tables.md [hunk 4]
```diff
@@ -84,6 +84,14 @@ Because the `test` database does not specify a [survival goal](multiregion-overv
 
 For more information about how to choose a database survival goal, see [When to Use `ZONE` vs. `REGION` Survival Goals](when-to-use-zone-vs-region-survival-goals.html).
 
+## Troubleshooting
+
+### High follower read latency on global tables
+
+reads on multi-region global tables can experience sporadic high latency on [follower reads](follower-reads.html) if the round trip time between cluster nodes is higher than 150ms. To work around this issue, consider setting the `kv.closed_timestamp.lead_for_global_reads_override` [cluster setting](cluster-settings.html) to a value greater than 800ms.
```
Comment (ianjevans) [inline v22.1/global-tables.md:91]:
```suggestion
Reads on multi-region global tables can experience sporadic high latency on [follower reads](follower-reads.html) if the round trip time between cluster nodes is higher than 150ms. To work around this issue, consider setting the `kv.closed_timestamp.lead_for_global_reads_override` [cluster setting](cluster-settings.html) to a value greater than 800ms. 
```

File: v22.1/global-tables.md [hunk 5]
```diff
@@ -84,6 +84,14 @@ Because the `test` database does not specify a [survival goal](multiregion-overv
 
 For more information about how to choose a database survival goal, see [When to Use `ZONE` vs. `REGION` Survival Goals](when-to-use-zone-vs-region-survival-goals.html).
 
+## Troubleshooting
+
+### High follower read latency on global tables
+
+reads on multi-region global tables can experience sporadic high latency on [follower reads](follower-reads.html) if the round trip time between cluster nodes is higher than 150ms. To work around this issue, consider setting the `kv.closed_timestamp.lead_for_global_reads_override` [cluster setting](cluster-settings.html) to a value greater than 800ms. 
+
+Note that this value will impact write latency to global tables, so you should proceed in 100ms increments until the high read latency no longer occurs. If you've increased the setting to 1500ms and the problem persists, you should [contact support](support-resources.html).
```
Comment (ianjevans) [inline v22.1/global-tables.md:93]:
```suggestion
The value of `kv.closed_timestamp.lead_for_global_reads_override` will impact write latency to global tables, so you should proceed in 100ms increments until the high read latency no longer occurs. If you've increased the setting to 1500ms and the problem persists, you should [contact support](support-resources.html).
```

File: v22.2/global-tables.md [hunk 6]
```diff
@@ -84,6 +84,14 @@ Because the `test` database does not specify a [survival goal](multiregion-overv
 
 For more information about how to choose a database survival goal, see [When to Use `ZONE` vs. `REGION` Survival Goals](when-to-use-zone-vs-region-survival-goals.html).
 
+## Troubleshooting
+
+### High follower read latency on global tables
+
+reads on multi-region global tables can experience sporadic high latency on [follower reads](follower-reads.html) if the round trip time between cluster nodes is higher than 150ms. To work around this issue, consider setting the `kv.closed_timestamp.lead_for_global_reads_override` [cluster setting](cluster-settings.html) to a value greater than 800ms.
```
Comment (ianjevans) [inline v22.2/global-tables.md:91]:
```suggestion
Reads on multi-region global tables can experience sporadic high latency on [follower reads](follower-reads.html) if the round trip time between cluster nodes is higher than 150ms. To work around this issue, consider setting the `kv.closed_timestamp.lead_for_global_reads_override` [cluster setting](cluster-settings.html) to a value greater than 800ms. 
```

File: v22.2/global-tables.md [hunk 7]
```diff
@@ -84,6 +84,14 @@ Because the `test` database does not specify a [survival goal](multiregion-overv
 
 For more information about how to choose a database survival goal, see [When to Use `ZONE` vs. `REGION` Survival Goals](when-to-use-zone-vs-region-survival-goals.html).
 
+## Troubleshooting
+
+### High follower read latency on global tables
+
+reads on multi-region global tables can experience sporadic high latency on [follower reads](follower-reads.html) if the round trip time between cluster nodes is higher than 150ms. To work around this issue, consider setting the `kv.closed_timestamp.lead_for_global_reads_override` [cluster setting](cluster-settings.html) to a value greater than 800ms. 
+
+Note that this value will impact write latency to global tables, so you should proceed in 100ms increments until the high read latency no longer occurs. If you've increased the setting to 1500ms and the problem persists, you should [contact support](support-resources.html).
```
Comment (ianjevans) [inline v22.2/global-tables.md:93]:
```suggestion
The value of `kv.closed_timestamp.lead_for_global_reads_override` will impact write latency to global tables, so you should proceed in 100ms increments until the high read latency no longer occurs. If you've increased the setting to 1500ms and the problem persists, you should [contact support](support-resources.html).
```

**Review by ianjevans — COMMENTED** [review]
Just a couple of nit fixes I made as suggestions.

File: v21.2/global-tables.md (hunk 2 above)
Comment (rmloveland) [reply v21.2/global-tables.md]: Fixed, thanks!

File: v21.2/global-tables.md (hunk 3 above)
Comment (rmloveland) [reply v21.2/global-tables.md]: Fixed, thanks!

File: v22.2/global-tables.md (hunk 6 above)
Comment (rmloveland) [reply v22.2/global-tables.md]: Fixed, thanks

File: v22.2/global-tables.md (hunk 7 above)
Comment (rmloveland) [reply v22.2/global-tables.md]: Fixed, thanks Ian!

**Pull request merged**